runtime/libc/rand.go

```diff
@@ -1,9 +1,14 @@
 package libc
 
-import "math/rand"
+import (
+	"math"
+	"math/rand"
+)
+
+const RandMax = math.MaxInt32-1
```

Comment:
I manually checked it with a simple program, it looks like `RAND_MAX` is exactly equal to `math.MaxInt32`.